hyperlink/_url.py

```diff
@@ -865,7 +849,7 @@ def absolute(self):
 
     def replace(self, scheme=_UNSET, host=_UNSET, path=_UNSET, query=_UNSET,
                 fragment=_UNSET, port=_UNSET, rooted=_UNSET, userinfo=_UNSET,
-                family=_UNSET, uses_netloc=_UNSET):
```

Comment:
Yeah, if it were me starting from scratch, I'd take everything after fragment and `**kw` it, I think. In this case I'm not sure how to rewrite it, except that it's probably for another PR :)